.travis.yml

```diff
@@ -61,26 +60,25 @@ before_install:
 - sudo apt-get install -y --force-yes r-base
 - R --version
 install:
-- if [[ $TRAVIS_SECURE_ENV_VARS == false && $CLOUD == mandatory ]]; then
+- if [[ $TRAVIS_SECURE_ENV_VARS == false && $TEST_TYPE == cloud ]]; then
     echo "Can't run cloud tests without keys so don't bother building";
   else
     ./gradlew assemble;
     ./gradlew installDist;
   fi
 script:
-#basic sanity test to check that gatk-launch actually runs something
-- ./gatk-launch PrintReads -I src/test/resources/NA12878.chr17_69k_70k.dictFix.bam -O output.bam
 #run tests
 #install git-lfs
-- if [[ $TRAVIS_SECURE_ENV_VARS == false && $CLOUD == mandatory ]]; then
+- if [[ $TRAVIS_SECURE_ENV_VARS == false && $TEST_TYPE == cloud ]]; then
     echo "Can't run cloud tests without keys so don't run tests";
   else
+    ./gatk-launch PrintReads -I src/test/resources/NA12878.chr17_69k_70k.dictFix.bam -O output.bam;
```

Comment:
Including the comment here broke travis linting.  I haven't figured out how to put comment lines in the middle of a bash statement in the travis script yet.  If you can figure it out, I'm happy to move it.  For now I'll just keep if above the block.